pkg/monitor/monitor/monitor_manager.go

```diff
@@ -157,31 +141,7 @@ func (m *MonitorManager) syncTidbMonitorService(monitor *v1alpha1.TidbMonitor) e
 	return nil
 }
 
-func (m *MonitorManager) syncTidbMonitorPVC(monitor *v1alpha1.TidbMonitor) (*corev1.PersistentVolumeClaim, error) {
-
-	pvc := getMonitorPVC(monitor)
-	pvc, err := m.deps.TypedControl.CreateOrUpdatePVC(monitor, pvc, false)
-	if err != nil {
-		klog.Errorf("tm[%s/%s]'s pvc failed to sync,err: %v", monitor.Namespace, monitor.Name, err)
-		return nil, err
-	}
-	return pvc, nil
-}
-
-func (m *MonitorManager) syncTidbMonitorPV(monitor *v1alpha1.TidbMonitor, pvc *corev1.PersistentVolumeClaim) error {
-	// update meta info for pv
-	pv, err := m.deps.PVLister.Get(pvc.Spec.VolumeName)
-	if err != nil {
-		return err
-	}
-	_, err = m.deps.PVControl.UpdateMetaInfo(monitor, pv)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-func (m *MonitorManager) syncTidbMonitorDeployment(tc *v1alpha1.TidbCluster, monitor *v1alpha1.TidbMonitor) error {
+func (m *MonitorManager) syncTidbMonitorStatefulset(tc *v1alpha1.TidbCluster, monitor *v1alpha1.TidbMonitor) error {
 
 	cm, err := m.syncTidbMonitorConfig(tc, monitor)
```

Comment:
The configmap name is fixed, so if we update the config in a new release, the  Pod will not be rolling updated, will this be addressed in another PR?

Reply:
Yes,another pr support it.